functions/http2/1.0/templayed.js

```diff
@@ -0,0 +1,137 @@
+// *
```

Comment:
Should this not be a shared directory, so it can be reused by other functions. so something in e.g. utils directory

Reply:
Maybe look at the expression step how it is done